pages.es/common/docker-container-top.md

```diff
@@ -0,0 +1,8 @@
+# docker container top
+
+> Este comando es un alias de `docker top`.
+> Más información: <https://docs.docker.com/reference/cli/docker/container/top/>.
+
+- Muestra la documentación del comando original:
```

Comment:
```suggestion
- Ver documentación para el comando original:
```

Reply:
I have used: Vea la documentación para el... in all the seven branches.  To make it more clear for the users that the docs refer to tldr docs that are produced for the original command. Using imperative move, to tell the user to do this action instead of describing what tldr actually does.

Thank you :)